### PR TITLE
br: explain more about restore speed in test report

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
explain more about restore speed.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13941
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-pitr-guide.md [hunk 1]
```diff
@@ -116,17 +116,25 @@ The following steps describe how to clean up backup data that exceeds the backup
 >
 > - Snapshot data restore speed = Snapshot data size / (duration * the number of TiKV nodes)
 > - Log data restore speed = Restored log data size / (duration * the number of TiKV nodes)
+>
+> The snapshot data size refers to the logical size of all KVs in a single replica, not the actual amount of restored data. BR restores all replicas according to the number of replicas configured for the cluster. If the number of replicas grows, the actual amount of restored data grows as well.
```
Comment (hfxsd) [inline br/br-pitr-guide.md:120]:
```suggestion
> The snapshot data size refers to the logical size of all KVs in a single replica, not the actual amount of restored data. BR restores all replicas according to the number of replicas configured for the cluster. The more replicas there are, the more data is actually restored.
```

File: br/br-pitr-guide.md [hunk 2]
```diff
@@ -116,17 +116,25 @@ The following steps describe how to clean up backup data that exceeds the backup
...
+> The default replica number for all clusters in the test is 3.
+> To improve the overall restore performance, you can modify the [`import.num-threads`](/tikv-configuration-file.md#import) item in the TiKV configuration file and the [`concurrency`](/br/use-br) option in the BR command.
```
Comment (hfxsd) [inline br/br-pitr-guide.md:122]:
```suggestion
> To improve the overall restore performance, you can modify the [`import.num-threads`](/tikv-configuration-file.md#import) item in the TiKV configuration file and the [`concurrency`](/br/use-br-command-line-tool.md#common-options) option in the BR command.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ A `br` command consists of multiple layers of sub-commands. Currently, br comman
 * `--cert`: specifies the path to the SSL certificate in the PEM format.
 * `--key`: specifies the path to the SSL certificate key in the PEM format.
 * `--status-addr`: specifies the listening address through which `br` provides statistics to Prometheus.
+* `--concurrency`: the number of concurrent tasks during the backup or recovery.
```
Comment (hfxsd) [inline br/use-br-command-line-tool.md:50]:
```suggestion
* `--concurrency`: the number of concurrent tasks during the backup or restore.
```

**Review by 3pointer — APPROVED** [review]

**Pull request merged**